Mani-Quotify/app/src/main/java/com/mani/quotify007/ui/screens/favorites/FavoritesScreen.kt

```diff
@@ -13,10 +13,16 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import com.mani.quotify007.domain.model.Quote
-import com.mani.quotify007.presentation.viewmodel.MainEvent
+import com.mani.quotify007.ui.navigation.model.MainEvent
+import com.mani.quotify007.ui.screens.quote.QuotesScreen
 
 @Composable
-fun FavoritesScreen(quotes: List<Quote>, onEvent: (MainEvent) -> Unit) {
+fun FavoritesScreen(
+    quotes: List<Quote>,
+    onEvent: (MainEvent) -> Unit,
+    onCopyText: (Quote) -> Unit,
+    onShareClick: (Quote) -> Unit
```

Comment:
if your functions are increasing, then can we have another approach?

Reply:
Will address this in my next PR